### PR TITLE
Placeholder Text for Search Boxes

### PR DESCRIPTION
**Pull request by krisklosterman**
I have many first time users who have no idea how to search because they could not find the search box

### CODE REVIEW

**Comment by krisklosterman** [comment]
66c9ac2 is an idea for the tabs and the search box

**Comment by krisklosterman** [comment]
Screen Shot 2019-04-17 at 9 18 56 PM
Screen Shot 2019-04-17 at 9 18 25 PM

File: src/Ombi/ClientApp/app/search/moviesearch.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <div role="tabpanel" class="tab-pane active" id="MoviesTab">
 
     <div class="input-group search-bar-background">
-        <input id="search" type="text" class="form-control form-control-custom form-control-search form-control-withbuttons"
+        <input id="search" type="text" placeholder="Type Here to Search" class="form-control form-control-custom form-control-search form-control-withbuttons"
```
Comment (tidusjar) [inline src/Ombi/ClientApp/app/search/moviesearch.component.html]: This ideally should have a translated value and not a hardcoded string
```suggestion
        <input id="search" type="text" placeholder="{{ 'Search.SearchBarPlaceholder' | translate}}" class="form-control form-control-custom form-control-search form-control-withbuttons"
```
You will need to create that in the `en.json` file

File: src/Ombi/ClientApp/app/search/music/musicsearch.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<!-- Movie tab -->
 <div role="tabpanel" class="tab-pane active" id="MoviesTab">
     <div class="input-group">
-        <input id="search" type="text" class="form-control form-control-custom form-control-search form-control-withbuttons" (keyup)="search($event)">
+        <input id="search" type="text" placeholder="Type Here to Search" class="form-control form-control-custom form-control-search form-control-withbuttons" (keyup)="search($event)">
```
Comment (tidusjar) [inline src/Ombi/ClientApp/app/search/music/musicsearch.component.html]: Same here regarding the placeholder translation

File: src/Ombi/ClientApp/app/search/tvsearch.component.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿<!-- Movie tab -->
 <div role="tabpanel" class="tab-pane" id="TvShowTab">
     <div class="input-group">
-        <input id="search" type="text" class="form-control form-control-custom form-control-search form-control-withbuttons" (keyup)="search($event)">
+        <input id="search" type="text" placeholder="Type Here to Search" class="form-control form-control-custom form-control-search form-control-withbuttons" (keyup)="search($event)">
```
Comment (tidusjar) [inline src/Ombi/ClientApp/app/search/tvsearch.component.html]: same here

File: src/Ombi/ClientApp/app/search/moviesearch.component.html (hunk 1 above)
Comment (krisklosterman) [reply src/Ombi/ClientApp/app/search/moviesearch.component.html]: Added

File: src/Ombi/ClientApp/app/search/music/musicsearch.component.html (hunk 2 above)
Comment (krisklosterman) [reply src/Ombi/ClientApp/app/search/music/musicsearch.component.html]: Added

File: src/Ombi/ClientApp/app/search/tvsearch.component.html (hunk 3 above)
Comment (krisklosterman) [reply src/Ombi/ClientApp/app/search/tvsearch.component.html]: Added

**Comment by tidusjar** [comment]
Nice work!

**Pull request merged**